terraform/aws/flink-operator-helm-values.yaml.tftpl

```diff
@@ -0,0 +1,30 @@
+defaultConfiguration:
+  flink-conf.yaml: |+
+    #### defaultConfiguration from pangeo-forge-cloud-federation ####
+    # on aws pull base image from ecr
+    kubernetes.container.image.ref: ${flink_image_registry}flink:${flink_version}
```

Comment:
nice find, didn't know about this. I wonder if this means we can leave this completely out of the runner or now if the runner overrides this value with deploy-specific precedence?

https://github.com/pangeo-forge/pangeo-forge-runner/blob/main/pangeo_forge_runner/bakery/flink.py#L247C9-L254

Reply:
Ouch, my spinning head. Fine to stew on this while Flink is on the shelf.

Reply:
Ah, but we do need to merge those other fixes to enable anything post `history-server` on USGS's cloud account.

Reply:
this should be fine b/c the deployment and `image:<>` tags override what is here but those won't be pointing to the ECR version. let's find out